### PR TITLE
Only show relevant extensions for sample template

### LINKED ISSUE #940: Sample template selection could only allow relevant file extensions
If the file extensions that are allowed for upload are restricted, we can internally make some choices about which files are suggested for processing sample templates (etc). Currently ``.fna`` files are suggested for this:

screenshot 2015-03-10 10 55 27

### PR DESCRIPTION
**Pull request by adamrp**
Fix #940 

Shows txt and tsv files. Any other extensions? I considered csv, but I don't think our parsers actually handle that format (and no one likes CSV files anyways, right?)

### CODE REVIEW

**Comment by ElDeveloper** [comment]
Can you add a text describing that only txt and tsv extensions are shown under that dropdown menu?

**Comment by ElDeveloper** [comment]
Also, this should also be added for the prep template.

**Comment by squirrelo** [comment]
:+1:  once timeout failure is done.

File: qiita_pet/templates/study_description_templates/raw_data_editor_tab.html [hunk 1]
```diff
@@ -12,10 +12,12 @@
         <ol>
           <!-- Step 1: choose the prep template file -->
           <li>
-            select your prep template file
+            select your prep template file (only files with the "txt" and "tsv" file extensions will be displayed here):
             <select id="add_prep_template_{{raw_data_id}}">
               {% for f in files %}
+                {% if f.endswith(('txt', 'tsv')) %}
```
Comment (wasade) [inline qiita_pet/templates/study_description_templates/raw_data_editor_tab.html]: did not know that was allowable...

**Comment by squirrelo** [comment]
Can you table-ize this? It's a bit ragge and messy-looking as is:
screen shot 2015-03-11 at 12 30 11 pm

File: qiita_pet/templates/study_description_templates/raw_data_editor_tab.html [hunk 2]
```diff
@@ -12,10 +12,12 @@
         <ol>
           <!-- Step 1: choose the prep template file -->
           <li>
-            select your prep template file
+            select your prep template file (only files with the "txt" and "tsv" file extensions will be displayed here):
```
Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/raw_data_editor_tab.html]: Can you do a `<br />` before the added text to break it up a bit? It's really long right now. Same for below.

File: qiita_pet/templates/study_description_templates/raw_data_editor_tab.html (hunk 1 above)
Comment (adamrp) [inline qiita_pet/templates/study_description_templates/raw_data_editor_tab.html]: I *think* I actually learned this from reviewing some of your code at some point last year, although I may be misremembering :laughing:

**Pull request merged**